src/components/Tag/Tag.scss

```
@@ -0,0 +1,18 @@
+@charset 'utf-8';
+@import '../../styles/partials/theme.scss';
+@import '../../styles/partials/mixin.scss';
+
+.tag {
+  position: absolute;
+  left: 20px;
+  top: 30px;
+  span {
+    padding: 5px 10px;
```

선택자 간에는 한 줄씩 띄워서 가독성을 높여 주세요!
```suggestion
.tag {
  position: absolute;
  left: 20px;
  top: 30px;
  
  span {
    padding: 5px 10px;
```